eclass/kernel-install.eclass

```diff
@@ -184,13 +184,28 @@ kernel-install_create_init() {
 
 	cat <<-_EOF_ >"${T}/init.c" || die
 		#include <stdio.h>
+		#include <sys/utsname.h>
+
+		int uname(struct utsname *buf);
```

Comment:
Why do you need to declare it if you included `sys/utsname.h` already?

Reply:
no need indeed.removed.